### PR TITLE
Push changes in the pkg-test

### PR DESCRIPTION
**Pull request by frogunder**

### CODE REVIEW

File: jenkins/setup_env_vars.py [hunk 1]
```diff
@@ -120,10 +125,13 @@ def set_env_vars(**kwargs):
     year = ver_split[0]
     month = ver_split[1]
     release = ver_split[2]
-    upgrade_from_version = year + '.' + month + '.' + str((int(release) -1))
-    if kwargs['salt_version'].split('.')[-1:][0] == '0':
-        upgrade_from_version = get_salt_version(kwargs['url'], kwargs['prev_branch'], prev_branch=True)
     branch = year + '.' + month
+#    upgrade_from_version = year + '.' + month + '.' + str((int(release) -1))
```
Comment (Ch3LL) [inline jenkins/setup_env_vars.py]: please remove the comment line

File: jenkins/setup_env_vars.py [hunk 2]
```diff
@@ -120,10 +125,13 @@ def set_env_vars(**kwargs):
...
+    if kwargs['salt_version'].split('.')[-1:][0] == '0':
+        upgrade_from_version = get_salt_version(kwargs['url'].replace('staging', ''), kwargs['prev_branch'], prev_branch=True)
+        #upgrade_from_version = get_salt_version(kwargs['url'], kwargs['prev_branch'], prev_branch=True)
```
Comment (Ch3LL) [inline jenkins/setup_env_vars.py]: here too

File: params.jinja [hunk 3]
```diff
@@ -55,7 +55,7 @@
 {% set python3 =  salt['pillar.get']('python3', False) %}
 {#{% set python3 = python3 if '2017' not in salt_version else False %}#}
 {% set python_major_version = '3' if python3 else '2' %}
-{% set python_version = 'python' %}
+{% set python_version = 'python3' %}
```
Comment (Ch3LL) [inline params.jinja]: What about python2? Why are we changing it to python3?

File: test_install/saltstack/debian.sls [hunk 4]
```diff
@@ -8,7 +8,7 @@
   {% set repo_url = 'https://repo.saltstack.com/{0}salt_rc/{1}/debian/{2}/{3}/' %}
   {% set repo_url = repo_url.format(params.dev, params.py_dir, params.os_major_release, params.os_arch) %}
 {% else %}
-  {% set repo_url = 'https://{0}repo.saltstack.com/{1}{2}/debian/{3}/{4}/archive/{5}' %}
+  {% set repo_url = 'https://{0}repo.saltstack.com/{1}{2}/debian/{3}/{4}/archive/{5}' %}
```
Comment (Ch3LL) [inline test_install/saltstack/debian.sls]:
```suggestion
  {% set repo_url = 'https://{0}repo.saltstack.com/{1}{2}/debian/{3}/{4}/archive/{5}' %}
```

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: jenkins/setup_env_vars.py (hunk 1 above)
Comment (frogunder) [reply jenkins/setup_env_vars.py]: removed

File: jenkins/setup_env_vars.py (hunk 2 above)
Comment (frogunder) [reply jenkins/setup_env_vars.py]: removed

File: params.jinja (hunk 3 above)
Comment (frogunder) [reply params.jinja]: Should only be python, changed it.

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**